Patch to add Solaris to the list of OSes which do not deliver mod-mpm.
This patch has not yet been submitted upstream.

--- puppetlabs-apache-1.4.0/manifests/mpm.pp.orig	2015-06-03 13:55:24.304854831 -0600
+++ puppetlabs-apache-1.4.0/manifests/mpm.pp	2015-06-03 13:55:33.301896584 -0600
@@ -65,7 +65,7 @@ define apache::mpm (
         mpm_module => $mpm
       }
     }
-    'redhat': {
+    'redhat', 'solaris': {
       # so we don't fail
     }
     default: {
